packages/nimble-components/src/table-column/text/tests/table-column-text.spec.ts

```diff
@@ -123,21 +123,60 @@ describe('TableColumnText', () => {
     });
 
     const fieldValues = [
-        { dataValue: 'foo', renderedValue: 'foo' },
-        { dataValue: '<button></button>', renderedValue: '<button></button>' },
-        { dataValue: 'null', renderedValue: 'null' },
-        { dataValue: 'undefined', renderedValue: 'undefined' }
+        { value: 'foo' },
+        { value: '<button></button>' },
+        { value: 'null' },
+        { value: 'undefined' },
+        { value: 'null' },
+        { value: '-2147483648/-1' },
+        { value: '田' },
+        { value: 'Ω' },
+        { value: '( ͡° ͜ʖ ͡°)' },
+        { value: '😍' },
+        { value: 'Iñtërnâtiônàlizætiøn☃💩' },
+        { value: '１' }
     ];
     for (const fieldValue of fieldValues) {
         // eslint-disable-next-line @typescript-eslint/no-loop-func
-        it(`data ${fieldValue.dataValue} renders as ${fieldValue.renderedValue}`, async () => {
+        it(`data "${fieldValue.value}" renders as "${fieldValue.value}"`, async () => {
```

Comment:
Can we use the getSpecTypeByNamedList helper to paramterize the tests? https://github.com/ni/nimble/blob/main/packages/nimble-components/src/utilities/tests/parameterized.ts#L58

In this case you just need to change the `value` field to be called `name`